### PR TITLE
Split Infiltrator class crate in two

### PR DESCRIPTION
**Pull request by Gannets**
## About the PR 
Splits the infiltrator class crate in to two seperate classes with different specialisations. Both recieve the regular syndicate specialist armour as well as...

**Infiltrator**

- tranq pistol + ammo pouch
- pod beacon deployer
- dna scrambler
- voice changer
- emag
- chameleon jumpsuit
- chameleon projector

**Scout**

- burst fire smg + ammo pouch
- cloaking device
- light breaker
- emag
- nvg

## Why's this needed? 
The current infiltrator kit isn't at all suited for infiltration missions. This PR allows players to pick between a kit that is adept at blending amongst the crew, taking out high value targets and collecting the disk and a kit that can sneakily attack the crew from their flanks, before cloaking to disengage.

## Changelog

```changelog
(u)Gannets
(*)Splits the Infiltrator nuclear operative class crate in to two kits. Infiltrator & Scout, details in minor changes.
(+)Infiltrator kit now comes with: tranq pistol + ammo pouch, pod beacon deployer, dna scrambler, voice changer, emag, chameleon jumpsuit, chameleon projector
(+)Scout kit comes with: burst fire smg + ammo pouch, light breaker, cloaking device, emag, nvg
```

### CODE REVIEW

**Comment by predmetparfuyit86** [comment]
whats a jumosuit

**Comment by flappybatpal** [comment]
Lightbreaker on scout could be interesting? You don't see much NVG use.

It may make it too easy for them to shoot people in the dark or cause issues for the rest of the nukies who rarely bring NVGs but it would really promote the flanking and fighting in the dark.

**Comment by Gannets** [comment]
Lightbreaker is a neat idea. Virva and I were also chatting about giving one of them the deployable pod beacon.

**Comment by BatElite** [comment]
I was curious if smoke grenades would be a decent idea for scouts.

**Comment by Gannets** [comment]
I'm going back and forth on that. Half tempted to remove grenade pouches from most kits to cut down on bulk. Maybe have them either on the cairngorm or as a utility.

File: code/obj/item/gun/kinetic.dm [hunk 1]
```diff
@@ -952,6 +952,31 @@ ABSTRACT_TYPE(/obj/item/gun/kinetic)
 		..()
 		ammo.amount_left = 0
 		update_icon()
+/obj/item/gun/kinetic/smg
+	name = "Bellatrix submachine gun"
+	desc = "A semi-automatic, 9mm submachine gun, developed by Almagest Weapons Fabrication."
+	icon = 'icons/obj/48x32.dmi'
+	icon_state = "mp52"
+	w_class = 2
```
Comment (Sovexe) [inline code/obj/item/gun/kinetic.dm:960]: this will fit into normal belts and boxes, is that desired?

Comment (Gannets) [reply code/obj/item/gun/kinetic.dm:960]: Hm, copy pasted from the pistol I bet. It should probably be a bit bigger.

**Pull request merged**